### PR TITLE
Fix some type of error

### PR DESCRIPTION
**Pull request by hhaAndroid**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Fix some type of error

### CODE REVIEW

File: mmdeploy/codebase/base/mmcodebase.py [hunk 1]
```diff
@@ -15,11 +16,11 @@ class MMCodebase(metaclass=ABCMeta):
 
     task_registry: Registry = None
 
-    def __init__() -> None:
+    def __init__(self) -> None:
         pass
 
     @classmethod
-    def get_task_class(cls, task: Task) -> type:
```
Comment (SingleZombie) [inline mmdeploy/codebase/base/mmcodebase.py:22]: Hi, I have a question here. The function will return a class. And the type of class is `type`. Is it ok to say the type of returned value is the class itself?

File: mmdeploy/utils/config_utils.py [hunk 2]
```diff
@@ -217,15 +217,16 @@ def cfg_apply_marks(deploy_cfg: Union[str, mmcv.Config]) -> Union[bool, None]:
     return apply_marks
 
 
-def get_partition_config(deploy_cfg: Union[str, mmcv.Config]) -> Dict:
+def get_partition_config(
+        deploy_cfg: Union[str, mmcv.Config]) -> Union[Dict, None]:
```
Comment (AllentDan) [inline mmdeploy/utils/config_utils.py:221]: `Union[Dict, None]` -> `Optional[Dict]`

**Comment by RunningLeon** [comment]
@hhaAndroid Hi, you may fix ci error.

**Comment by hhaAndroid** [comment]
> @hhaAndroid Hi, you may fix ci error.

OK

File: mmdeploy/utils/config_utils.py [hunk 3]
```diff
@@ -252,7 +253,7 @@ def get_calib_config(deploy_cfg: Union[str, mmcv.Config]) -> Dict:
     return calib_config
 
 
-def get_calib_filename(deploy_cfg: Union[str, mmcv.Config]) -> str:
+def get_calib_filename(deploy_cfg: Union[str, mmcv.Config]) -> Optional[str]:
```
Comment (AllentDan) [inline mmdeploy/utils/config_utils.py:256]: Hi, @hhaAndroid. May update the docstring as well

**Review by AllentDan — APPROVED** [review]
LGTM

**Review by SingleZombie — APPROVED** [review]

**Pull request merged**